src/js/modules/userAuth/userData.js

```diff
@@ -25,15 +25,6 @@ const fetchOrderCountAndSave = userData => {
             method: 'GET',
             credentials: 'same-origin'
         })
-            .then(response => {
-                if (response.ok) {
```

Comment:
So, just so we record something around this, the issue with this block was that we couldn't get the unit tests to fall into this branch? Would be good to look into this a little more when work on the `f-fetch` kicks off as it looks like something which we'd likely need to be able to test against. 

Also, it looks like this change means that we are no longer passing the `statusText` along with the thrown error, was there a specific reason why were doing that in the first place?